autoPyTorch/utils/pipeline.py

```diff
@@ -23,75 +22,61 @@
 
 
 def get_dataset_requirements(info: Dict[str, Any],
-                             include_estimators: Optional[List[str]] = None,
-                             exclude_estimators: Optional[List[str]] = None,
-                             include_preprocessors: Optional[List[str]] = None,
-                             exclude_preprocessors: Optional[List[str]] = None
+                             include: Optional[Dict] = None,
```

Comment:
What kind of change is it?
Could you describe what changed from the previous parameters?

Reply:
The previous parameters are not compatible or needed for autoPyTorch, I think we left it unchanged from when we used the code from autosklearn. In our case, we need to pass 4 things to instantiate the pipeline
1. Dataset properties
2. include
3. exclude
4. search space updates.

Reply:
Is it possible to change the name include or exclude to include_something and exclude_something?

Plus I did not get from what we will include or exclude the preprocessor in the previous code. So it will be helpful if you could write the description for them.

Reply:
I'll add a doc string but include and exclude is also how its called in our pipelines, So to change that I suggest you to raise an issue and we can address in a different PR.